ledger-bridge/index.js

```diff
@@ -0,0 +1,46 @@
+import PubNub from 'pubnub';
```

Comment:
@mvines, do you want JS and associated dependencies in this git repository? Seems like it belongs elsewhere. Maybe in solana-web3.js?

Reply:
Keeping the js separate for now is my preference, possibly living in the git repo that hosts the  block explorer source.

Reply:
I like that. So maybe (if replicator API is the future), this becomes a JS bridge to the replicator API that can stream to different sinks PubNub, Redis, etc.?